docs/spelling_wordlist.txt

```diff
@@ -20,7 +21,10 @@ Copr
 dataset
 datasets
 Debian
+debuggability
+dedup
 Deis
+dev
```

Comment:
This seems to be the only instance of this word in this change, so it can be removed.